cmd/create/kubeletconfig/cmd.go

```diff
@@ -93,19 +93,15 @@ func run(_ *cobra.Command, _ []string) {
 		r.Reporter.Infof("Enabling interactive mode")
 	}
 
+	maxPidsLimit, err := GetMaxPidsLimit(r.OCMClient)
```

Comment:
Could It be beneficial to put the changes in this file into a helper file considering both create and edit are using the same exact code?

Reply:
@den-rgb Let me see what I can do 👍

Reply:
@den-rgb Refactored as per your suggestion into the `ValidateOrPromptForRequestedPidsLimit` function. Looks much cleaner, so thanks for that!

cheers,
Rob